hedera-node/hedera-file-service-impl/src/main/resources/genesis/throttles-dev.json

```diff
@@ -70,8 +70,9 @@
 			"TokenUnfreezeAccount",
 			"TokenGrantKycToAccount",
 			"TokenRevokeKycFromAccount",
-			"TokenUpdateNfts",
-			"TokenReject"
+			  "TokenUpdateNfts",
+			  "TokenAirdrop",
```

Comment:
I would assume `TokenAirdrops` should be in same throttles OpsPerSec as `CryptoTransfer`